pymunk/tests/test_space.py

```diff
@@ -167,15 +165,6 @@ def pre_solve_remove(arb: p.Arbiter, space: p.Space, data: Any) -> bool:
 
         s.step(0.1)
         return
-        self.assertTrue(b in s.bodies)
```

Comment:
These lines were left for future use as a reminder / comment.. so not for removal. Same with the other similar cases